about/index.html

```diff
@@ -0,0 +1,257 @@
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            <div class="col-md-9 office">
+                <div class="row justify-content-around my-2 mb-5">
+                    <div class="col-md-5 col-lg-3">
+                        <img src="/about/imgs/placeholder.png" class="img-fluid" alt="placeholder 1">
+                        <h4 class="text-primary mt-1">Mr. Gary Sequeira</h4>
+                        <h6 class="text-dark">Chairperson</h6>
+                    </div>
+                    <div class="col-md-5 col-lg-3">
+                        <img src="/about/imgs/twishisaran.jpg" class="img-fluid" alt="placeholder 1">
+                        <h4 class="text-primary mt-1">Ms. Twishi Saran</h4>
+                        <h6 class="text-dark">Vice-Chairperson</h6>
+                    </div>            
+                    <div class="col-md-5 col-lg-3">
+                        <img src="/about/imgs/placeholder.png" class="img-fluid" alt="placeholder 1">
+                        <h4 class="text-primary mt-1">Mr. Varad Narvekar</h4>
+                        <h6 class="text-dark">Secretary</h6>
+                    </div>
+                    <div class="col-md-5 col-lg-3">
+                        <img src="/about/imgs/placeholder.png" class="img-fluid" alt="placeholder 1">
+                        <h4 class="text-primary mt-1">Mr. Vidyasagar Mekkalki</h4>
+                        <h6 class="text-dark">Treasurer</h6>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    
+    <!-- IEEE GEC WIE Chapter Content -->
+
+    <div class="container-fluid">
+
+            <div class="row text-center border-bottom mb-5 pb-3">
+
+                <div class="col-md-3">
+                    <div class="row my-2 justify-content-center">
+                        <div class="card" style="width: 18rem;">
+                            <img src="/about/imgs/ieeesb.jpg" class="card-img-top" alt="IEEE GEC WIE Logo">
+                            <div class="card-body">
+                                <h5 class="card-title text-primary">IEEE GEC WIE Council</h5>
+                                <p class="card-text">PLACEHOLDER TEXT<br>Established on ... <br>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minus nam provident ab. Magnam, necessitatibus ratione?</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            <div class="col-md-9 office">
+                <div class="row justify-content-around my-2 mb-5">
+                    <div class="col-md-5 col-lg-3">
+                        <img src="/about/imgs/siyakamat.jpg" class="img-fluid" alt="placeholder 1">
```

Comment:
This file is HUGE!!! Crop and compress it.

Reply:
If cropping can't be possible, just compress it. https://tinyjpg.com/ does the work well